tests/Unit/DataStructures/Test_ScaleBy.cpp

```diff
@@ -0,0 +1,87 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/Tensor/EagerMath/ScaleBy.hpp"
+
+TEST_CASE("Unit.DataStructures.Tensor.ScaleBy", "[Functors][Unit]") {
+  const size_t npts = 8;
+  DataVector one(npts, 1.0);
+  DataVector two(npts, 2.0);
+  DataVector three(npts, 3.0);
+  DataVector four(npts, 4.0);
+  DataVector five(npts, 5.0);
+  DataVector twelve(npts, 12.0);
+  DataVector thirteen(npts, 13.0);
+
+  Tensor<DataVector, Symmetry<1>,
```

Comment:
There is a constructor for `tnsr::vector` that takes a list of `DataVector`s that could be used here. You may need three braces, unfortunately, to get it to work.